hw_diag/utilities/shell.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import os
+
+
+def get_environment_var(diagnostics):
+    # The order of the values in the lists is important!
+    # It determines which value will be available for which key
+    env_var = [
+        'BALENA_DEVICE_NAME_AT_INIT',
+        'BALENA_DEVICE_UUID',
+        'BALENA_APP_NAME',
+        'FREQ',
+        'FIRMWARE_VERSION',
+        'VARIANT'
+    ]
+    keys = ["BN", "ID", "BA", "FR", "FW", "VA"]
+
+    for (var, key) in zip(env_var, keys):
```

Comment:
We should probably create a dictionary and iterate over that

env_var_mappings = {
  'BALENA_DEVICE_NAME_AT_INIT': 'BN',
  ...
}

But really, why not just use the env var name as the key on diagnostics? I made the same comment earlier. Is there a benefit in renaming the keys to be shorter?

Reply:
This code is lifted from the rework that Ilya completed previously. Not sure if there is any benefit in using a hash table over using 2 lists and the zip function, I didn't really feel the need to change it.

We cannot alter the keys as we need to retain backwards compatibility, a requirement from Viktor, as the factory tooling relies on this naming convention. Renaming these keys should be completed though, as they are ugly right now, but this is probably out of scope for a Flask conversion and needs to be a story for the future so we can ensure the factory tooling is updated.